app/controllers/explorer.js

```diff
@@ -38,7 +38,7 @@ export default class ExplorerController extends Controller {
   // The comparison geography ID.
   // Must match ID of one option in comparisonGeoOptions.
   // Default "0" maps to NYC.
-  @tracked compareTo = "0";
+  @tracked compareTo = "1";
```

Comment:
Curious why this change? Don't we still want it to default to 0 so that the default comparison is citywide?

Reply:
In prod we're defaulting to Manhattan as the default comparison so I changed it here in for consistency. This initial state actually wasn't previously being used until you changed the comparison geography since the Explorer tab would link directly to /explorer/cities/NYC?**compareTo=1**

Reply:
Where do you see it defaulting to Manhattan? I picked a random census tract and clicked "View Data" in prod and it looks like it's comparing to citywide
<img width="1477" alt="image" src="https://user-images.githubusercontent.com/9055367/181281947-325edbe3-50e7-4455-b1e6-c5271ced87fc.png">

Reply:
oooh I see what's going on. In prod, we default the _comparison area_ to citywide if the user picks a geography on the map and then clicks "View Data" to reach the Explorer. If the user just clicks "Data Explorer" in the nav menu, they have citywide as their _selection_ and Manhattan as the comparison area. This change seems to make Manhattan the default comparison for users that get to the Explorer via "View Data", which I don't think we want. I think the logic here is the default comparison should be citywide, unless the selection is citywide, then its Manhattan